packages/assistify/package.js

```diff
@@ -17,6 +17,7 @@ Package.onUse(function(api) {
 
 	//Server
 	api.addFiles('config.js', 'server');
+	//TODO add jquery?
```

Comment:
@tkurz: already undestood. In my reply, you can find the declaration of dependency which will make sure jquery is available.